gcp_variant_transforms/libs/processed_variant_test.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from __future__ import absolute_import
+
```

Comment:
line.

Reply:
No, here line apparently was missing, so needs to stay.